configuration/prod/validator_1.toml

```diff
@@ -0,0 +1,32 @@
+server_config_path = "server_1.json"
+host = "34.151.215.158"
+port = 9100
+internal_host = "validator-1"
+internal_port = 10100
+external_protocol = { Simple = "Tcp" }
+internal_protocol = { Simple = "Tcp" }
+
+# If deploying to GCP, host should be the Kubernetes pod name
+[[shards]]
+host = "server-1-shard-0"
+port = 9100
```

Comment:
9100 seems to be the default Prometheus Node Exporter port.
This might be a good opportunity to change our ports, also for the local validator config files.